mlora/MixLoRA.py

```diff
@@ -0,0 +1,133 @@
+from mlora.modelargs import MultiLoraBatchData
+from mlora.LoraLiner import Linear
+
+import torch
+import torch.nn.functional as F
+from typing import Dict, Tuple, Optional
+
+
+class MixGate(torch.nn.Module):
```

Comment:
it is a independent NN model or just the gate for Mix-LORA

Reply:
MixGate can be regarded as a MoE Config. It is not only the Gate layer, but also handles MoE related logic, such as routing.